samples/hello_world/src/main.c

```diff
@@ -6,8 +6,14 @@
 
 #include <zephyr/kernel.h>
 
+/* delay between greetings (in ms) */
+#define SLEEPTIME 1000
+
 int main(void)
 {
-	printk("Hello World! %s\n", CONFIG_BOARD);
+	while(1) {
+		printk("Hello World! %s\n", CONFIG_BOARD);
+		k_msleep(SLEEPTIME);
```

Comment:
```suggestion
		k_msleep(1000);
```

Reply:
just for simplicity, also, I'd change `printk` to `printf`, and include `<stdio.h>`